apps/notifications/src/panel/templates/container-hotkey.jsx

```diff
@@ -31,7 +31,7 @@ export class HotkeyContainer extends Component {
 	}
 
 	handleKeyDown = ( event ) => {
-		if ( ! this.props.shortcuts || ! this.props.keyboardShortcutsAreEnabled ) {
+		if ( ! this.props.shortcuts || false === this.props.keyboardShortcutsAreEnabled ) {
```

Comment:
What prompted this change? Upon inspecting this looks fine though, since state initializes this with the value `false` and only toggles between the actual boolean values there.

Reply:
Reverted this change - it was done to allow the i shortcut key to work on main panel but I didn't need it after I enabled the shortcuts in the `ShortcutsPopover` component.